src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -825,95 +825,124 @@ void makeMemberMaybeNull(MethodSymbol method, string memberName)
                 }
             }
 
-            void enforceNotNullWhenForPendingReturn(PendingBranch pendingReturn, BoundReturnStatement returnStatement)
```

Comment:
:memo: The local functions were extracted without modification (except for naming changes)